src/script.c

```diff
@@ -528,6 +538,10 @@ void scriptCall(scriptRunCtx *run_ctx, robj* *argv, int argc, sds *err) {
         goto error;
     }
 
+    if (scriptVerifyMayReplicate(run_ctx, err) != C_OK) {
+        return;
```

Comment:
```suggestion
        goto error;
```
So we will count the command as rejected?

Reply:
thx, fixed